src/razor/src/razorLanguageServerOptionsResolver.ts

```diff
@@ -23,17 +23,7 @@ export function resolveRazorLanguageServerOptions(
     const usingOmniSharp =
         !getCSharpDevKit() && vscodeApi.workspace.getConfiguration().get<boolean>('dotnet.server.useOmnisharp');
 
-    const hotReload = vscodeApi.workspace.getConfiguration('csharp.experimental.debug').get<boolean>('hotReload');
```

Comment:
If someone had set the fuse setting to false, would changing the default value of it cause their setting to change? If not, we may still need this??

Reply:
> If someone had set the fuse setting to false

This doesn't override their setting. It just sets it to true if the value was unset (null default). If they set to false it should be equivalent today. We have telemetry for that so as we roll this out we can see if we need to change the name to override that (or just remove the option altogether)

Reply:
Oh you're right, sorry. I thought this made the hot reload setting override the FUSE setting, but it is only when unset. My b.